source/Halibut/Transport/SecureWebSocketListener.cs

```diff
@@ -86,12 +84,12 @@ async Task Accept()
                 {
                     try
                     {
-                        var context = await listener.GetContextAsync();
+                        var context = await listener.GetContextAsync().ConfigureAwait(false);
 
                         if (context.Request.IsWebSocketRequest)
                         {
 #pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
```

Comment:
Is this still relevant? I saw it disappear in `SecureListener.cs `, so slight question about the difference

Reply:
It is, because we're in an `async Task` method here, whereas in the equivalent location in `SecureListener.cs` the containing method is just `void`.

Reply:
We could make `SecureListener` async... but I don't want to poke the bear right now.